packages/gasket-cli/src/commands/create.js

```diff
@@ -61,7 +62,17 @@ class CreateCommand extends Command {
         await loadPreset(context);
         cliVersion(context);
         applyPresetConfig(context);
-        await globalPrompts(context);
```

Comment:
Per other comment, I think you should still fire this action, even if `--config` is provided. It'll handle all the wiring up so you don't have to duplicate logic. Just be sure to read the config before.